### ISSUE TITLE
Accessibility of error text in ScaleTextField

### ISSUE THREAD

**Issue by tschinski** [telekom/scale#1446]
Hi guys,

we're facing an accessibility issue with ScaleTextField component.
Unfortunately the error text of an ScaleTextField is not recognized by screen readers. 

https://telekom.github.io/scale/?path=/docs/components-text-field--with-error

Actually, all kind of helper texts are not recognized.

Do we miss something?

Thanks in advance and best regards
Tobias

**Comment by marvinLaubenstein** [comment]
Hi @tschinski, sorry that I have not answered for so long. The accessibility errors and/or deficits should be fixed in the near future.
